### ISSUE TITLE
[question] order of columns in export

### ISSUE THREAD

**Issue by machytkafitanalytics** [lukasmartinelli/pgclimb#6]
Hi, looks like you always export columns in alphabetical order (jsonline format). Would it be possible to preserve order in table or to add some switch into command line? Thanks. Josef

**Comment by cbertelli** [comment]
Yes, I see the same behaviour and it happens even if my select has an explicit order of columns. Could you  please provide a parameter or switch that prevents this in JSON and JSONLINEs?

**Comment by cbertelli** [comment]
BTW, if the user wants to reorder columns, she may decide to use an ordered list of columns in the (mandatory) SQL query, no need to force the JSON to a different order.
It could be useful to omit null columns (with a parameter) as complained in #18 because JSON already has a structure.
